XamlControlsGallery/ControlPages/FlipViewPage.xaml

```diff
@@ -32,7 +32,7 @@
 &lt;FlipView MaxWidth="400" Height="270"&gt;
     &lt;Image Source="ms-appx:///Assets/SampleMedia/cliff.jpg" AutomationProperties.Name="Cliff"/&gt;
     &lt;Image Source="ms-appx:///Assets/SampleMedia/grapes.jpg" AutomationProperties.Name="Grapes"/&gt;
-    &lt;Image Source="ms-appx:///Assets/SampleMedia/rainer.jpg" AutomationProperties.Name="Rainier"/&gt;
+    &lt;Image Source="ms-appx:///Assets/SampleMedia/rainier.jpg" AutomationProperties.Name="Rainier"/&gt;
```

Comment:
Doesn't this break the FlipView page?

Reply:
The change was to the sample code. The code that gets executed had the correct spelling already.